decorator.rb

```diff
@@ -0,0 +1,27 @@
+class Decorator < Nameable
+  def initialize(nameable)
+    super()
+    @nameable = nameable
+  end
```

Comment:
- [ ] Please add getters and setters `attr_accessor :nameable` to make the decorator class have access to it.

Reply:
- [ ] [REQUIRED] Kindly ensure that the `nameable` class is accessible by the `decorator` class by adding `attr_accessor :nameable` as requested by the last reviewer.

Reply:
PS: Hello, next reviewer! This has been resolved and it is ready for approval 👍🏽